Documentation/tutorial/walkthrough_create_a_docfx_project.md

```diff
@@ -0,0 +1,183 @@
+        {
+            inner.Name = name;
+            _class = inner;
+        }
+
+        public class InnerClass
+        {
+            public string Name { get; set; }
+        }
+    }
+}
+```
+
+2. Call `docfx metadata` under `D:\docfx_walkthrough`. `docfx metadata` is a sub-command registered in `docfx`, it reads configuration in the `metadata` section from `docfx.json`. `[ "src/**.csproj" ]` in `metadata/src/files` tells `docfx` to search all the `csproj` from `src` subfolder  to generate metadata.
+
+```json
+"metadata": [
+{
+    "src": [
+    {
+        "files": [ "src/**.csproj" ],
+        "exclude": [ "**/bin/**", "**/obj/**" ]
+    }
+    ],
+    "dest": "obj/api"
+}
+]
+```
+
+3. `obj/api` folder is generated. There are several `yml` files generated under the folder. It contains the data model extracted from c# source code file. YAML is the metadata format used in `docfx`. [General Metadata Spec](http://dotnet.github.io/docfx/spec/metadata_format_spec.html) defines the general schema and [.NET Metadata Spec](http://dotnet.github.io/docfx/spec/metadata_dotnet_spec.html) defines the metadata schema for .NET languages that `docfx` can be consumed.
+```
+|- HelloDocfx.Class1.InnerClass.yml
+|- HelloDocfx.Class1.yml
+|- HelloDocfx.yml
+|- toc.yml
+```
+
+Step4. `build` part: generate a simple documentation website
+----------------------------------------------------
+Now we can generate a simple website based on the `yml` files just created.
+1. Still under `D:\docfx_walkthrough`, create a new file naming `toc.yml` with the following content:
+
+```yml
+- name: About
+  href: index.md
+- name: API reference
+  href: obj/api/
+```
+
+2. create a new file naming `index.md` and add some content to it. `index.md` will be the homepage for our website.
+
+```md
+Welcome to the world of docfx!
+=============================
+
+Hello *everyone*.
+```
+
+2. Run command `docfx build`. Note that a new subfolder `_site` is generated under the same folder. This is where the static website is generated.
+3. Run command `docfx serve _site`. If port `8080` is not in use, `docfx` will host `_site` under `http://localhost:8080`. If `8080` is in use, you can use `docfx serve _site -p <port>` to change the port to be used by `docfx`.
+4. Start a browser and navigate to `http://localhost:8080` and the following website will show. Note that the navbar is exactly what we set in `toc.yml` before.
+![homepage](images/walkthrough_homepage.png)
+5. Navigate to `http://localhost:8080/api/HelloDocfx.html` and the following website will show. Note that the this page contains the API generated in *Step3*.
+![api page](images/walkthrough_api.png)
+
+Step5. 🔧 Advanced: Add more tabs to navbar
+----------------------------------------
+> **Note**
+> 
+> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+
```

Comment:
Shoud we move "Advanced" to a new article? A shorter walkthrough may be friendly for users. we can congratulate user here, then provide a link to "Advanced".